src/CalculatorUnitTests/HistoryTests.cpp

```diff
@@ -46,17 +46,16 @@ namespace CalculatorFunctionalTests
         {
             m_standardViewModel = ref new StandardCalculatorViewModel();
             m_standardViewModel->IsStandard = true;
-            m_historyViewModel = ref new HistoryViewModel(&(m_standardViewModel->m_standardCalculatorManager));
-            m_historyViewModel->SetCalculatorDisplay(m_standardViewModel->m_calculatorDisplay);
```

Comment:
Did you mean to remove this line too?

Reply:
yes, `HistoryVM` in `StandardViewModel` is already initialized line 117 of `StandardCalculatorViewModel`